bob-apps/bin/bobp.rs

```diff
@@ -649,42 +750,23 @@ fn print_periodic_stat(
             } else {
                 first_line = false;
             }
-
-            let d_get = get_count.get_diff();
-            let get_count_spd = d_get * 1000 / period_ms;
-            let cur_st_get_time = get_time_st.get_diff() as f64;
-            let cur_st_get_count = get_count_st.get_diff() as f64;
-            let get_error = stat.get_error_count.load(Ordering::Relaxed);
-            let get_spd = get_size.get_diff() as f64 / 1024. / sec;
-            println!(
-                "get:   {:>6} rps | err {:5} | {:>6.2} kb/s | lat {:>6.2} ms",
-                get_count_spd,
-                get_error,
-                get_spd,
-                finite_or_default(cur_st_get_time / cur_st_get_count / 1e9)
-            );
-
+            let get_spd = periodic_get.process_and_print();
             get_speed_values.push(get_spd);
         }
+        if print_delete {
+            if !first_line {
+                print!("{:>6}", ' ');
+            } else {
+                first_line = false;
+            }
```

Comment:
Declare `print_offset` function and use it for every operation: `first_line = print_offset(first_line);`